src/packaging/packages.targets

```diff
@@ -60,14 +60,37 @@
             <ILCompilerSdkFiles Include="bootstrappercpp" />
             <ILCompilerSdkFiles Include="System.Private.CoreLib.Native" Condition="'$(OsEnvironment)'!='Windows_NT'" />
 
+            <!-- ILCompiler.SDK Cpp Codegen support files -->
+            <ILCompilerSdkCppCodegenFiles Include="Native/gc/env/etmdummy.h" />
+            <ILCompilerSdkCppCodegenFiles Include="Native/gc/env/gcenv.base.h" />
```

Comment:
I think this list of files will need fixing up before we actually start using them instead the stale copy from appdeps package (e.g. I think we will need `gcenv.structs.h` and `sal.h`).

I have done cleanup around this in  #557. We should only need `gcenv.structs.h` and `gcenv.base.h` in the SDK from under Native/GC after this PR.